frontend/src/components/dialogs/GDialog.vue

```diff
@@ -37,7 +37,6 @@ limitations under the License.
       <v-alert tile :color="confirmAlertColor" v-if="confirmValue && !confirmDisabled">
         <span class="text-body-2" v-if="!!confirmMessage">{{confirmMessage}}</span>
         <v-text-field
```

Comment:
I saw this error a couple of times already in the log. Do you know why we had this here and did this even work at some time?

Reply:
Yes, initially we had a ConfirmInputDialog with a button
`<v-btn @click.native.stop="okClicked()" :disabled="confirmDisabled || hasError" class="red--text" flat>Confirm</v-btn>`

and having an ok and cancel function passed as prop.
```js
    props: {
      value: {
        type: Boolean,
        required: true
      },
      ok: {
        type: Function,
        required: true
      },
      cancel: {
        type: Function,
        required: true
      },
      ..
```
which was called in the okClicked / cancelClicked function
```js
    methods: {
      cancelClicked () {
        if (this.cancel) {
          this.cancel()
        }
      },
      okClicked () {
        if (this.ok) {
          this.ok()
        }
      },
```

at some point in time we agreed that passing a function as prop is not a good idea, hence it was removed..

Reply:
ah okay. So now the component that is using the dialog must take over this functionality (if required) by listening to the key events? Then I guess we can approve this. We should however align the general behavior of dialogs. I think some of them can be confirmed via keys while others do not allow this.

Reply:
Good point. I now resolve the action if it is valid. This should restore the old behavior
For other dialogs, we should discuss. E.g. for the create cluster dialog I do not think it's a good idea to "create" on enter press

Reply:
I should have noticed this already in the first review.